src/main/java/org/opentripplanner/standalone/config/routerequest/RouteRequestConfig.java

```diff
@@ -335,176 +336,52 @@ The board time is added to the time when going from the stop (offboard) to onboa
 
   private static void mapBikePreferences(NodeAdapter c, BikePreferences.Builder builder) {
     var dft = builder.original();
+    NodeAdapter cb = c.of("bicycle").since(V2_5).summary("Bicycle preferences.").asObject();
```

Comment:
I would call the `cb` just c and the c `root`. This would then follow the pattern used elsewhere. We pass in `root` only because we need to, and the node we are interested in is the `c`.